src/storage/src/hummock/sstable/group_builder.rs

```diff
@@ -123,13 +127,15 @@ where
         grouping: G,
         policy: CachePolicy,
         sstable_store: SstableStoreRef,
+        uploading_sender: Option<Arc<mpsc::UnboundedSender<UploadRequest>>>,
```

Comment:
We don't need `Arc` for the sender. We can just clone it if we want to have more than one sender.

Reply:
> We don't need `Arc` for the sender. We can just clone it if we want to have more than one sender.

we want to destroy sender automatically

Reply:
> We don't need `Arc` for the sender. We can just clone it if we want to have more than one sender.

in some occasion we want to pass `sender` with small cost

Reply:
`Sender` itself already implements reference counting internally, and its `Clone` implementation is basically the same as `Arc` clone. So I think we can use `sender.clone()`.

I have submitted a new PR to remove the `Arc` of sender introduced in the PR mentioned above. #3289

Reply:
> `Sender` itself already implements reference counting internally, and its `Clone` implementation is basically the same as `Arc` clone. So I think we can use `sender.clone()`.
> 
> I have submitted a new PR to remove the `Arc` of sender introduced in the PR mentioned above. #3289

fixed

Reply:
> We don't need `Arc` for the sender. We can just clone it if we want to have more than one sender.

just removed that